sample.yml

```diff
@@ -4,6 +4,19 @@
 # on components in your cluster!
 - include: "{{ playbook_dir }}/playbooks/check-requirements.yml"
 
+# ENABLE REPOS - all of our packages are installed from these repos, we need to
+# enable them before any roles can be used.
+- hosts: all
+  tasks:
+    - name: enable yum repos
+      sudo: yes
+      copy:
+        src: "roles/common/files/{{ item }}"
+        dest: "/etc/yum.repos.d/{{ item }}"
+      with_items:
+        - asteris-mantl-rpm.repo
+        - ciscocloud-rpm.repo
+
 # BASICS - we need every node in the cluster to have common software running to
```

Comment:
I'm just wondering if this is the best place for this. A fresh install failed for me b/c I hadn't update my local mantl.yml with this task from sample.yml. I'm not sure how widespread of an issue this will be since it won't affect new users but it might bite more of us who have existing Mantl configs. And, in general, keeping the sample.yml as simple as possible seems like a good thing. Do you think it might be cleaner to just put this in a tiny role that is included as a dependency in common?